src/Database/Schema/TableSchema.php

```diff
@@ -456,7 +456,7 @@ public function defaultValues(): array
      * {@inheritDoc}
      * @throws \Cake\Database\Exception
      */
-    public function addIndex(string $name, array $attrs): TableSchemaInterface
+    public function addIndex(string $name, $attrs): TableSchemaInterface
```

Comment:
Should we instead remove the dead string option here?

Reply:
There are no tests for using string value for `$attrs` but code does have `is_string()` check.

We could remove passing string if other's agree.

Reply:
We can add it then to migration guide.